composer.json

```diff
@@ -32,8 +32,7 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^5",
-        "prestashop/php-dev-tools": "^2"
```

Comment:
So, we no longer require phpstan for this module :thinking:

Reply:
Same for cs fixer :sweat_smile:

Reply:
Have you tried the 3.x of php-dev-tools ? Tools and configurations files have been updated (btw the bootstrap.php could be removed from the module)

Reply:
`prestashop/php-dev-tools` was never installed in fact because it requires a more recent version of `symfony/filesystem` than the one installed. Also, phpstan requires php 7.1 and we are setting php 5.6 as minimum version for this module.
I think that whatever choice we make, it should probably be done in another PR as it will be more than just fix the tests

Reply:
Weird because i'm able to install it with https://github.com/PrestaShop/ps_facetedsearch/blob/dev/composer.json#L17 even if the requirement is PHP 5.6 +

Reply:
Oh! I think it's because in autoupgrade the requirement is `"php": "5.6"` when it's `"php": ">=5.6"` in ps_facetedsearch. I'll have a look ;)
But I'm a bit confused too as @Progi1984 told me explicitly to not use `php-dev-tools` because it was buggy 🤪 but I guess it's OK now!

Reply:
@PierreRambaud I just checked and `prestashop/php-dev-tools` doesn't require `phpstan` so that's why you can install it in `ps_facetedsearch` but `prestashop/php-dev-tools` requires `symfony/filesystem` >= 3.2 while we're using version 2.8 here. I think we should keep this PR as this and do a new PR to upgrade the `symfony/filesystem` dependency and install `prestashop/php-dev-tools` as well. WDYT?

Reply:
Yes, improve it in a new PR :)